jujugui/static/gui/src/app/utils/statsd.js

```diff
@@ -19,24 +19,25 @@ const StatsClient = class StatsClient {
   */
   constructor(url, prefix='', flags={}) {
     this.url = url.replace(/\/?$/, '/');
-    this.flags = flags;
+    this._flags = flags || {};
```

Comment:
Yes indeed, but now flags are provided as an argument, and some sanity check could maybe help?